### PR TITLE
chore: genereate railroad diagrams

### PR DESCRIPTION
**Pull request by Yaiba**
This add railroad diagrams to parse package

### CODE REVIEW

File: parse/grammar/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+## Railroad Diagrams
+Diagrams are generated by [ebnf-convert](https://github.com/GuntherRademacher/ebnf-convert) + [rr](https://github.com/GuntherRademacher/rr).
```
Comment (Yaiba) [inline parse/grammar/README.md:3]: We can make this automatic in a github action, not worth it right now

**Comment by brennanjl** [comment]
lgtm, thank you

**Review by brennanjl — APPROVED** [review]

**Comment by jchappelow** [comment]
Looking through the changes, I do not think this warrants a new tag for the `parse` module.  
The links to these diagrams can point to the main module tag, i.e. `v0.8.1`.
Sound good @Yaiba @brennanjl ?

**Comment by Yaiba** [comment]
> Looking through the changes, I do not think this warrants a new tag for the `parse` module. The links to these diagrams can point to the main module tag, i.e. `v0.8.1`. Sound good @Yaiba @brennanjl ?

yeah totally not worth a tag.

As this is merged to main, diagrams are available at  https://kwilteam.github.io/kwil-db/parse/grammar/rrdiagrams.html @luisgruson @KwilLuke

**Pull request merged**